doc/data/messages/r/return-or-yield-in-finally/bad.py

```diff
@@ -0,0 +1,20 @@
+def second_favorite():
```

Comment:
It's possible to create multiple files in a directory too, for multiple examples (a bad directory and a good directory) I think this would be better. We've never done it before so we'll have to deal with the surprises if they are any :)

Reply:
Ok, let's see what happens :grinning:

Reply:
https://pylint--8440.org.readthedocs.build/en/8440/user_guide/messages/warning/return-or-yield-in-finally.html

:champagne: